tokio/src/lib.rs

```diff
@@ -347,6 +347,8 @@ cfg_time! {
     pub mod time;
 }
 
+pub mod syscall;
```

Comment:
should the module be private when the syscall feature isn't enabled? that's what we tend to do with other modules that are feature-flagged but used internally...

Reply:
Yes that makes sense, we can make this `pub(crate)` when the `syscall` flag is disabled.